front/app/modules/commercial/smart_groups/containers/RulesGroupFormWithValidation/RulesGroupForm.tsx

```diff
@@ -27,68 +35,97 @@ const SSectionField = styled(SectionField)`
 import { Multiloc } from 'typings';
 import { TRule } from '../../components/UserFilterConditions/rules';
 
-export interface Props {}
 export interface RulesFormValues {
   rules: TRule[];
   title_multiloc: Multiloc;
-  membership_type: 'rules';
+  membership_type: 'rules' | 'manual';
 }
 
-export class RulesGroupForm extends React.PureComponent<
-  InjectedFormikProps<Props, RulesFormValues>
-> {
-  render() {
-    const { isSubmitting, errors, touched, status } = this.props;
+type Props = {
+  onSubmit: (formValues: RulesFormValues) => void | Promise<void>;
+  defaultValues?: Partial<RulesFormValues>;
+  isVerificationEnabled: boolean;
+} & InjectedIntlProps;
+
+const RulesGroupForm = ({
+  intl: { formatMessage },
+  onSubmit,
+  defaultValues,
+  isVerificationEnabled,
+}: Props) => {
+  const schema = object({
+    title_multiloc: validateMultiloc(formatMessage(messages.titleFieldError)),
```

Comment:
Since smart groups are only ever used for admins and never exposed to citizens (at least right now), I find it's needless and potentially annoying to demand they enter the group name in all languages. The FE validation we had before (and the BE validation) only required one language to be provided, which I think is a better UX in this case.

Reply:
I totally agree in this case, it doesn't make sense to require all languages! I implemented it in this way for this form. 
We also discussed creating a reusable utility for this but I have my doubts about that. The reason is that in the vast majority of cases, it's beneficial to ensure a value is entered for all languages (off the top of my head - topics, navbar items, etc.) and even though the backend validation requires only one language for many of these, it's reasonable to have a stricter rule on the front-end. I also don't want to introduce too many validation-related utilities as our setup is already complex enough as it is. Let me know if you disagree / feel strongly that validation should be consistent of both sides in all cases.